feconf.py

```diff
@@ -619,6 +619,7 @@ def get_empty_ratings():
 EXPLORATION_DATA_PREFIX = '/createhandler/data'
 EXPLORATION_FEATURES_PREFIX = '/explorehandler/features'
 EXPLORATION_INIT_URL_PREFIX = '/explorehandler/init'
+EXPLORATION_LEARNER_ANSWER_DETAILS = '/createhandler/learner_answer_details'
```

Comment:
is this going to be more generic in the future? (For exploration and question) Or we are going to create a different handler for the question part?

Also, through this url, we are going to create `learner_answer_details` or view `learner_answer_details`?

Reply:
For question, there will be a different handler for viewing the learner answer details. And this URL is to view learner_answer_details.

Reply:
If it is to view, don't have it be /createhandler, have some other name.

Reply:
Done